assets/scripts/im.poi.js

```diff
@@ -70,6 +70,18 @@ define('im.poi', ['im.leafletWrapper', 'im.config', 'im.renderUI', 'im.i18n'], f
 		popup = createPoiPopup(poi, popupWidth);
 		marker.bindPopup(popup);
 
+		// Temporary change required for ad purpose - https://wikia-inc.atlassian.net/browse/DAT-4051
+		// Open popups on hover on maps related to the ad campaign
+		// TODO: remove this as a part of https://wikia-inc.atlassian.net/browse/DAT-4055
+		if (config.mapConfig.isProtected) {
+			marker.on('mouseover', function () {
+				this.openPopup();
```

Comment:
This is still WIP and I have issues with smoke testing so I'll let you know ones I could try it out. This  PR is just an initial round of CR